calyx-py/calyx/py_ast.py

```diff
@@ -362,6 +376,10 @@ class Atom(GuardExpr):
     def doc(self) -> str:
         return self.item.doc()
 
+    @property
+    def name(self) -> str:
+        return f"{self.item.get_name()}"
+
```

Comment:
@anshumanmohan [not urgent] Requested a re-review to ask if you could glance at this `get_name` stuff when you get a chance? It was surprisingly annoying to get right.

Reply:
Aw man yeah it does look rough. Thanks for doing it, the case statements do look much more readable at least to my eye!